Movied/Movied/Scenes/Home/HomeViewController.swift

```diff
@@ -1,9 +1,215 @@
 import UIKit
 
 final class HomeViewController: UIViewController {
-
+    @IBOutlet private weak var homeTitle: UILabel!
+    @IBOutlet private weak var movieListCollectionView: UICollectionView!
+    @IBOutlet private weak var sortedByTextField: UITextField!
+    
+    private var allMovies: [Movie] = []
+    private var sortType: SortType = .newestToOldest
+    private var pageNumber = 1
+    
+    private let refreshControl = UIRefreshControl()
+    private let pickerView = UIPickerView()
+    private var refreshFooterView: RefreshFooterView?
+    
     override func viewDidLoad() {
         super.viewDidLoad()
+        showIndicator(seconds: 2)
```

Comment:
why call this command? it's fake and incorrect logic.
The correct way is show loading when start request api, and dismiss loading when the fetching data is completed or the error happened